docs/root/modules/aws/schema.md

```diff
@@ -192,7 +192,11 @@ Representation of an AWS [Inspector Finding](https://docs.aws.amazon.com/inspect
     ```
     (AWSInspectorFinding)-[:AFFECTS]->(ECRImage)
     ```
+- AWSInspectorFinding may be delegated from AWSAccounts.
```

Comment:
Can we add a detail that describes how this is different from the :RESOURCE relationship?

Reply:
Sure, let me add the reference regarding delegated accounts and their findings.

Reply:
While trying to document this clearly I noticed that `DELEGATE` was not a great word to describe the relationship and moved it to `MEMBER` since it is closer to the definition from AWS: https://docs.aws.amazon.com/inspector/latest/user/admin-member-relationship.html
> The delegated administrator can also view and retrieve information for their own account and any associated member. 
...
> A member account can view and retrieve information about their account in Amazon Inspector, while settings for their account are managed by the delegated administrator.